Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp

```diff
@@ -894,8 +894,8 @@ void JitArm64::subfex(UGeckoInstruction inst)
     ARM64Reg WA = gpr.GetReg();
     if (js.carryFlagSet)
     {
-      MOVI2R(WA, ~i + j, gpr.R(d));
-      ADC(gpr.R(d), WA, WZR);
+      MOVI2R(WA, ~i + j, static_cast<bool>(gpr.R(d)));
```

Comment:
Bug. The third parameter should just be removed

Reply:
Done.